schemas/host.yml

```diff
@@ -17,6 +17,14 @@
       description: >
         Hostname of the host.
 
+        It normally contains what the hostname command returns on the host machine.
```

Comment:
Nit: quote `hostname` like in the `.name` description

Reply:
applied